modules/postgresql/main.tf

```diff
@@ -67,12 +67,13 @@ resource "google_sql_database_instance" "default" {
   instance_type        = local.is_secondary_instance ? "READ_REPLICA_INSTANCE" : var.instance_type
 
   settings {
-    tier                        = var.tier
-    edition                     = var.edition
-    activation_policy           = var.activation_policy
-    availability_type           = var.availability_type
-    deletion_protection_enabled = var.deletion_protection_enabled
-    connector_enforcement       = local.connector_enforcement
+    tier                         = var.tier
+    edition                      = var.edition
+    activation_policy            = var.activation_policy
+    availability_type            = var.availability_type
+    deletion_protection_enabled  = var.deletion_protection_enabled
+    connector_enforcement        = local.connector_enforcement
+    enable_google_ml_integration = var.enable_google_ml_integration
```

Comment:
This was added in [5.25](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2d5b1154fa6f5450c3bea2d47398f94150196f08/modules/postgresql/versions.tf#L30) so let's bump min versions

https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2d5b1154fa6f5450c3bea2d47398f94150196f08/modules/postgresql/versions.tf#L30

Reply:
Done.